tests/e2e/default/transitions/create-user-for-contacts.replace-user.wdio-spec.js

```diff
@@ -781,6 +831,8 @@ describe('Create user for contacts', () => {
         const basicReportId3 = await submitBasicForm();
         const subsequentBasicReports = await chtDbUtils.getDocs([basicReportId2, basicReportId3]);
         subsequentBasicReports.forEach((report) => expect(report.contact._id).to.equal(originalContactId));
+
+        await assertFeedbackDocs();
```

Comment:
It's not unrelated. 
Because we recalculate in the background now, sync will happen outside of the test, and it generated feedback docs once the user became "disabled" server-side.